tests/e2e/tensor_ops/BUILD.bazel

```diff
@@ -112,24 +81,18 @@ iree_check_single_backend_test_suite(
 
 iree_check_single_backend_test_suite(
     name = "check_vulkan-spirv_vulkan",
-    srcs = enforce_glob(
-        # keep sorted
-        [
-            "collapse_shape.mlir",
-            "concat.mlir",
-            "expand_shape.mlir",
-            "extract_slice.mlir",
-            "tensor_cast.mlir",
-            "tensor_insert_slice.mlir",
-        ],
-        include = ["*.mlir"],
-        exclude = [
-            "pack.mlir",
-            "pack_dynamic_inner_tiles.mlir",
-            "pack_i8.mlir",
```

Comment:
I name it with `vnni` because of the inner tile sizes. They are the i8 pack ops we are seeing on CPU path (with vnni features). For other backends, they will likely see other pack ops, and they don't have to support vnni cases. It's a plus to handle it for functionality complement, but they don't really care about these pack ops.

The current status is: The vulkan-spirv backend does not support pack op codegen at this moment. I recently added the support for rocm side. CPU should be able to codegen all the pack ops.

Reply:
Okay, thanks. How about I move `tests/e2e/tensor_ops/pack_i8.mlir` to `tests/e2e/cpu_specific/pack_i8_vnni.mlir`, following the model of `tests/e2e/vulkan_specific/`?

Reply:
SGTM, thanks for cleaning it up! I totally forgot that there is `tests/e2e/cpu_specific`.

Reply:
There isn't, I'd be creating it for this :P

Reply:
I forgot that you removed it (https://github.com/iree-org/iree/commit/f3e9839ee5326ed984f418c1cd7662b21ed91131) long time ago. :p

Reply:
around and around we go :D